README

```diff
@@ -107,6 +108,7 @@ Note: the `-c` flag sets CPU affinity. If running both commands on same host, ma
 ## Our Community
 
 * [Project Website](http://www.openucx.org/)
+* [ReadTheDocs](https://openucx.readthedocs.io/en/master/)
```

Comment:
shall we also update links to `https://github.com/openucx/ucx/wiki` to point to RTD?

Reply:
I think, yes. Are you okay with this changes ?

Reply:
yes

Reply:
pls add review approval